spec/requests/authorization_spec.rb

```diff
@@ -1,5 +1,42 @@
 require "rails_helper"
 
+RSpec.shared_examples "subscriber and unsubscriber" do
```

Comment:
Isn't this already covered in the *_subscriber_spec.rb specs? It doesn't seem like much of this belongs in an authorization spec.

Reply:
so I found that when i wanted to test that search can only be used caseworker that none of the other authorization rules were explcitly being tested, so i went off on a tangent of excercising the subsccriber authorization rules from an integration spec point of view as a starting point. I was going to add for submissions as well before i pulled myself out of this tangent, and just tested the search rule i had added.

happy to remove but it feels like we do not actually integration test the authorization rules, despite coverage being happy. What say you?

Reply:
Fair play, happy to leave this in.